scripts/symcheck.py

```
@@ -0,0 +1,322 @@
+    ]
+
+
+def run_javap(flags, root, names):
+    """javap over every class at once. Batched because per-class spawning dominates
+    the runtime on a JAR with a few hundred classes."""
+    if not names:
+        return ""
+    out = []
+    # Keep argv under typical limits.
+    for i in range(0, len(names), 400):
+        r = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)